Meme/Meme/Models/Meme.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  Meme.swift
+//  Meme
+//
+//  Created by Mesfin Bekele Mekonnen on 7/29/19.
+//  Copyright © 2019 Mesfin Bekele Mekonnen. All rights reserved.
+//
+
+import Foundation
+
+struct Meme {
+    let name: String
+    let imageURL: URL
```

Comment:
I can't help but feel like `Template` could encapsulate this since they're really not that different, but Ive made you change so much already in this PR - we can hold off for the next one.

Reply:
Haha, yeah you're right. Can address in the next PR.